ambari/datadog_checks/ambari/data/conf.yaml.example

```diff
@@ -119,6 +119,30 @@ instances:
 #
 # logs:
 #   - type: file
+#     path: /var/log/ambari-server/ambari-alerts.log
+#     source: ambari
+#     service: ambari
+#     log_processing_rules:
+#        - type: multi_line
+#          name: new_log_start_with_date
+#          pattern: \d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])  # 2019-04-22 15:47:00,999
+#   - type: file
+#     path: /var/log/ambari-server/ambari-audit.log
+#     source: ambari
+#     service: ambari
+#     log_processing_rules:
+#        - type: multi_line
+#          name: new_log_start_with_date
+#          pattern: \d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])  # 2019-05-25T13:13:17.875Z
+#   - type: file
+#     path: /var/log/ambari-server/ambari-server-command.log
+#     source: ambari
+#     service: ambari
+#     log_processing_rules:
+#        - type: multi_line
+#          name: new_log_start_with_date
+#          pattern: (INFO)|(ERROR)|(WARN)  # ERROR 2019-04-22
+#   - type: file
 #     path: /var/log/ambari-server/*.log
```

Comment:
The assumption here is this will match the rest of file logs, is that right @NBParis ?

Reply:
That's a good question. @ajacquemot do you know if we apply the configuration in the sequential order? 

Like is there a change that the multiline configuration would be overridden by the wildcard one here?

Reply:
If the config are in the same file, there should be guarantee on the order but I think it's worth checking.

Reply:
@ajacquemot What do you mean by same file?

Reply:
the same yaml configuration file. Like it's not a race condition from configuration in different yaml files so the order should be properly respected.

The best way to confirm is to make a test though. I can help with that if necessary.

Reply:
AFAIK there is only one config file per integration but if there are doubts about this I'd rather list the remaining log files

Reply:
If there are not too many (less than 2), let's list them. That will be safer.

Reply:
Yup, so I did